WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1140,6 +1144,20 @@ public void initializeEditorFragment() {
             );
             aztecEditorFragment.setAztecVideoLoader(new AztecVideoLoader(getBaseContext(), loadingVideoPlaceholder));
             aztecEditorFragment.setLoadingVideoPlaceholder(loadingVideoPlaceholder);
+
+            if (getSite() != null && getSite().isWPCom() && !getSite().isPrivate()) {
+                // Add the content reporting for wpcom blogs that are not private
+                aztecEditorFragment.enableContentLogOnCrashes(
+                        new AztecExceptionHandler.ExceptionHandlerHelper() {
+                            @Override
+                            public boolean shouldLog(Throwable throwable) {
+                                // Do not log private or password protected post
+                                return getPost() != null && TextUtils.isEmpty(getPost().getPassword()) &&
```

Comment:
shouldn't this check be **either** password protected posts, **or** private posts? 

like this: (haven't tested)
```
return getPost() != null && (TextUtils.isEmpty(getPost().getPassword()) ||
!"private".equals(getPost().getStatus()));
```

Reply:
Ohh right! 🤦‍♂️

Reply:
Sorry Mario, the `if` condition seems OK to me. Both of the three checks should be true in order to log the content. So the post should be not null, it should not have a password, and its status should not be `private.

Reply:
Can't we have a `public` post that is password-protected? Otherwise, if only `private` posts can be password-protected, then the only condition to be checked should be `!"private".equals(getPost().getStatus()`

Reply:
Yes, you can set a password on public post, but we don't want to log that content, since it's private. You can't read on Internet without the password.

Reply:
understood - 👍